extensions/NOname-awa/More comparisons

```diff
@@ -0,0 +1,750 @@
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_DECIMAL_NUMBER',
+                                    default: 'decimal number'
+                                }),
+                                value: 'decimalNumber'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_PRIME_NUMBER',
+                                    default: 'prime number'
+                                }),
+                                value: 'primeNumber'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_COMPOSITE_NUMBER',
+                                    default: 'composite number'
+                                }),
+                                value: 'compositeNumber'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_TEXT',
+                                    default: 'text'
+                                }),
+                                value: 'text'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_BOOLEAN',
+                                    default: 'boolean'
+                                }),
+                                value: 'boolean'
+                            }
+                        ]
+                    }
+                }
+            };
+        }
+        booleanMenu({ BOOLEAN }) {
+            if (BOOLEAN === 'random') return Math.random() < 0.5 ? true : false
+            return BOOLEAN == 'true' ? true : false;
+        }
+        isTrue({ OPERAND }) {
+            return Cast.toBoolean(OPERAND);
+        }
+        checkType({ INPUT, MODE }) {
+            INPUT = String(INPUT);
+            const isNumRegex = /^-?\d+(\.\d+)?$/;
+            const isPrime = (number) => {
+                if (number <= 1) return false;
+                if (number <= 3) return true;
+                if (number % 2 === 0 || number % 3 === 0) return false;
+                let i = 5;
+                while (i * i <= number) {
+                    if (number % i === 0 || number % (i + 2) === 0) return false;
+                    i += 6;
+                }
+                return true;
+            }
+            const isComposite = (number) => {
+                if (number <= 1) return false;
+                for (let i = 2; i <= Math.sqrt(number); i++) {
+                    if (number % i === 0) return true;
+                }
+                return false;
+            }
+            switch (MODE) {
```

Comment:
And you're positive this switch will always return _something_, even without a default?